PeerioTechnologies/peerio-desktop
# fix: AK visibility toggle errors

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/15267/desktop-console-errors-when-clicking-hide-show-ak-button-on-login

#### Testing instructions  
On login, AK visibility toggle should no longer produce console errors, and should focus AK text input when clicked.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
